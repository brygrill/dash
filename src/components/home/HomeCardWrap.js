// @flow
import React, { Component } from 'react';

export default class AppFooter extends Component {
  props: {
    children: any,
  };

  render() {
    return (
      <div className="mdl-cell mdl-cell--4-col-desktop mdl-cell--6-col-tablet mdl-cell--1m-offset-tablet mdl-cell--12-col mdl-cell--middle sbr-home-individual-card">
        {this.props.children}
      </div>
    );
  }
}
