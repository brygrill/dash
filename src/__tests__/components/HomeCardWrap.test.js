import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import HomeCardWrap from '../../components/home/HomeCardWrap';

describe('<HomeCardWrap />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MuiThemeProvider><HomeCardWrap /></MuiThemeProvider>, div);
  });
  it('Matches snapshot with child', () => {
    const component = renderer.create(
      <MuiThemeProvider><HomeCardWrap /></MuiThemeProvider>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
