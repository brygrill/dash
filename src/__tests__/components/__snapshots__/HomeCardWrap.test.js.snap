// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeCardWrap /> Matches snapshot with child 1`] = `
<div
  className="mdl-cell mdl-cell--4-col-desktop mdl-cell--6-col-tablet mdl-cell--1m-offset-tablet mdl-cell--12-col mdl-cell--middle sbr-home-individual-card"
/>
`;
